meta-trust-m/recipes-client/trust-m/trust-m_0.1.0.2.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Install the trust-m tooling"
+DESCRIPTION = "Install the trust-m tooling"
+LICENSE = "MIT"
+
+SRCREV = "${AUTOREV}"
+PV = "1.0+git${SRCPV}"
+SRC_URI:aarch64 = "gitsm://github.com/Infineon/linux-optiga-trust-m.git;protocol=https;branch=development_v3"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=fc1a86d5b2e9230b6e718647d5ea1252"
```

Comment:
It's interesting to know `md5` is used for the license file checksum.

Reply:
I know, long time no `md5` right ! The docs I checked might be outdated, I'll see if they support `sha256sum` in newer versions.